blazar/plugins/oshosts/host_plugin.py

```diff
@@ -564,6 +564,7 @@ def list_allocations(self, query, detail=False):
         hosts_allocations = self.query_host_allocations(hosts_id_list,
                                                         **options)
         self.add_extra_allocation_info(hosts_allocations)
+        self.add_allocation_cleaning_time(hosts_allocations, CONF.cleaning_time)
```

Comment:
again checking understanding:

there's a prior commit that enables "enforcing" cleaning time when attempting to make new leases
and the issue was that when users/calendar viewed existing ones, the valid times now didn't match up.

and so this pr adds the cleaning time to the "list allocations" api response, so that it matches the cleaning time setting.

Is there a "get allocation" api we also need to change?

Reply:
Fixed for get allocation. This hasn't really ever been used due to a bug fixed in https://github.com/ChameleonCloud/python-blazarclient/pull/38